frontend/src/components/pages/topics/Tab.Messages/index.tsx

```diff
@@ -417,30 +403,38 @@ export class TopicMessageView extends Component<TopicMessageViewProps> {
                     */}
 
                     {/* Filter Tags */}
-                    {searchParams.filtersEnabled && (
-                        <div style={{ paddingTop: '1em', width: '100%' }}>
-                            <MessageSearchFilterBar messageSearch={this.messageSearch} />
-                        </div>
-                    )}
-
-                    {/* Show warning if a deserializer is forced for key or value */}
-                    {isDeserializerOverrideActive && (
-                        <Flex alignItems="flex-end" height="32px" width="100%" gap="4">
-                            {isKeyDeserializerActive && <Tag>
-                                <TagLabel cursor="pointer" onClick={() => this.showColumnSettings = true}>
-                                    Key Deserializer = {proto3.getEnumType(PayloadEncoding).findNumber(uiState.topicSettings.keyDeserializer)?.localName}
-                                </TagLabel>
-                                <TagCloseButton onClick={() => uiState.topicSettings.keyDeserializer = PayloadEncoding.UNSPECIFIED} />
-                            </Tag>}
-                            {isValueDeserializerActive && <Tag>
-                                <TagLabel cursor="pointer" onClick={() => this.showColumnSettings = true}>
-                                    Value Deserializer = {proto3.getEnumType(PayloadEncoding).findNumber(uiState.topicSettings.valueDeserializer)?.localName}
-                                </TagLabel>
-                                <TagCloseButton onClick={() => uiState.topicSettings.valueDeserializer = PayloadEncoding.UNSPECIFIED} />
-                            </Tag>}
-                        </Flex>
-                    )}
-                </Flex>
+                    {!isServerless() && <MessageSearchFilterBar
+                        messageSearch={this.messageSearch}
+                        canUseFilters={canUseFilters}
+                        menuEl={menuEl}
+                    />}
```

Comment:
Before this was `searchParams.filtersEnabled && (` now it is `!isServerless() && <...` 
Is this a mistake? Or just me not understanding it? :D

Reply:
Nice catch, I've also removed `filtersEnabled` from the file as it's always enabled now.